package.json

```diff
@@ -103,13 +103,16 @@
     "jest": "^27.0.4",
     "lint-staged": "^11.0.0",
     "marked": "^2.0.1",
+    "postcss": "^8.3.6",
     "prettier": "^2.2.1",
     "rimraf": "^3.0.2",
+    "rollup": "^2.56.3",
     "superagent": "^6.1.0",
     "typescript": "~4.3.0",
     "vite": "^2.1.3",
     "vue": "^3.2.1",
-    "vue-router": "^4.0.5"
+    "vue-router": "^4.0.5",
+    "webpack": "^5.51.1"
```

Comment:
webpack is not mean to be a dev dep. it is only needed when user want to use playground (i want them to install webpack manually, since it is not small)

Reply:
This is reasonable, let's see if we can make it optional.

Reply:
Well, this is not possible, but I'm ok with one more dev dep, comparing to the annoying warning message from the package manager.

Reply:
playground is not for daily developing but for experiments. adding webpack to dev dep wont make examples inside playground work out of box. so just remove it, for people
who want to do experiments, they can install webpack or
 express manually.

Reply:
Webpack is not required for component developing.

Reply:
That's fair.